### DISCUSSION TITLE
How do I edit a script?

### DISCUSSION THREAD

**Discussion by pkulak** [requarks/wiki#5077] [Help / Questions] [answered]
This sounds silly, but I created a page with a script, and I can't figure out how to modify the script after the fact, apart from recreating the page. Thanks for any help!

**Comment by NGPixel** [comment, marked as answer]
While editing a page, click on **Page** in the top-right corner, then on **Scripts**.

**Reply by pkulak** [reply]
lol, thank you! No idea how I missed that. :D
